src/chicory/backend/redis.py

```
@@ -73,7 +74,7 @@ async def store_result(self, task_id: str, result: TaskResult[Any]) -> None:
 
     async def get_result(self, task_id: str) -> TaskResult[Any] | None:
         if not self._client:
-            raise RuntimeError("Backend not connected")
+            raise RuntimeError("Backend nƒot connected")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)